recipe/build.sh

```diff
@@ -25,15 +29,6 @@ CPPFLAGS="-I$PREFIX/include" LDFLAGS="-L$PREFIX/lib" \
 make
 make install
 
-if [ $(uname) == Darwin ]; then
-    # Copy TIFF and GeoTIFF Headers to build against internal libraries.
-    mkdir -p $PREFIX/include/gdal/frmts/gtiff/libgeotiff
-    cp frmts/gtiff/libgeotiff/*.h $PREFIX/include/gdal/frmts/gtiff/libgeotiff
-    cp frmts/gtiff/libgeotiff/*.inc $PREFIX/include/gdal/frmts/gtiff/libgeotiff
-    mkdir -p $PREFIX/include/gdal/frmts/gtiff/libtiff
-    cp frmts/gtiff/libtiff/*.h $PREFIX/include/gdal/frmts/gtiff/libtiff
-fi
-
```

Comment:
:+1: 
Can we add a test to check if the tiff functionalities of `gdal` are there?